qutip/core/operators.py

```diff
@@ -471,6 +472,90 @@ def create(N, offset=0, *, dtype=None):
     return qdiags(data, -1, dtype=dtype)
 
 
+def fdestroy(N, site=0):
+    """
+    Fermionic destruction operator.
+    We use the Jordan-Wigner transformation,
+    making use of the so-called Jordan-Wigner ZZ..Z strings,
+    to construct this as follows (in Latex):
+    a_j = sigma_z^{otimes j}
+          otimes (frac{sigma_x + i sigma_y}{2})
+          otimes I^{otimes N-j-1}
+
+    Parameters
+    ----------
+    N : int
+        Dimension of Hilbert space.
```

Comment:
This is no longer the dimension of the space, but the number of sites. Maybe one should use a different parameter here instead of `N` to avoid confusion.

Reply:
good idea, I changed `N` to `n_sites`